package.json

```
@@ -15,6 +15,7 @@
     "lint:types": "pnpm --filter '*' lint:types",
     "prepare": "pnpm build",
     "postinstall": "lefthook install",
+    "release": "pnpm release-it",
```

minor, but since it's added as a script and targeting a binary, I don't think `pnpm` is needed here

```suggestion
    "release": "release-it",
```

---

Good catch, updated!